src/domain/cache/index.ts

```diff
@@ -3,6 +3,7 @@ export * from "./errors"
 export const CacheKeys = {
   CurrentPrice: "price:current",
   PriceHistory: "price:history",
+  LastOnChainTransactions: "bitcoin:lastOnChainTxs",
```

Comment:
it's only mempool tx right? maybe this should be more evident by the name

Reply:
nevermind I read back the code

Reply:
not only mempool, lnd return confirmed and not confirmed transactions